cl/validator/committee_subscription/committee_subscription.go

```diff
@@ -40,7 +40,7 @@ type CommitteeSubscribeMgmt struct {
 	// subscriptions
 	aggregationPool    aggregation.AggregationPool
 	validatorSubsMutex sync.RWMutex
-	validatorSubs      map[uint64]map[uint64]*validatorSub // slot -> committeeIndex -> validatorSub
```

Comment:
curious why don't need slot check anymore.
and i guess we don't need to record `validatorIdxs` in `validatorSub`, right? if so i can remove it and simplify this module in another pr.

Reply:
ok handled it